sources/packages/web/src/services/http/StudentApi.ts

```diff
@@ -62,4 +63,13 @@ export class StudentApi extends HttpBaseClient {
       throw error;
     }
   }
+
+  public async getProgramYears(): Promise<ProgramYear[]> {
+    const response = await this.apiClient.get(
+      "students/program-year",
```

Comment:
The program year is not a student-reated concept, it broader than (IMO). I would recommend moving it to a dedicated controller.

Reply:
Restructured